### PR TITLE
US038 - Add a linting check (backend)

### PR DESCRIPTION
**Pull request by ManpreetM01**

### CODE REVIEW

**Review by LMBishop — APPROVED** [review]

File: src/main/java/org/kainos/ea/DropwizardWebServiceConfiguration.java [hunk 1]
```diff
@@ -1,9 +1,8 @@
 package org.kainos.ea;
 
 import io.dropwizard.Configuration;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import org.hibernate.validator.constraints.*;
-import javax.validation.constraints.*;
+//import com.fasterxml.jackson.annotation.JsonProperty;
```
Comment (TomBillingtonUK) [inline src/main/java/org/kainos/ea/DropwizardWebServiceConfiguration.java]: Please delete lines instead of commeting them out

File: README.md [hunk 2]
```diff
@@ -10,4 +10,5 @@ How to start the DropwizardWebService application
 Health Check
 ---
 
-To see your applications health enter url `http://localhost:8081/healthcheck`
\ No newline at end of file
+To see your applications health enter url `http://localhost:8081/healthcheck`
```
Comment (TomBillingtonUK) [inline README.md:13]: Can you add how to run lint to the Readme

File: src/main/java/org/kainos/ea/DropwizardWebServiceConfiguration.java (hunk 1 above)
Comment (ManpreetM01) [reply src/main/java/org/kainos/ea/DropwizardWebServiceConfiguration.java]: change made

File: README.md (hunk 2 above)
Comment (ManpreetM01) [reply README.md:13]: change made

**Review by TomBillingtonUK — APPROVED** [review]

**Pull request merged**